osu.Game/Database/ArchiveModelManager.cs

```diff
@@ -183,16 +190,17 @@ protected async Task<IEnumerable<TModel>> Import(ProgressNotification notificati
 
         /// <summary>
         /// Import one <typeparamref name="TModel"/> from the filesystem and delete the file on success.
+        /// Note that this bypasses the UI flow and should only be used for special cases or testing.
         /// </summary>
-        /// <param name="path">The archive location on disk.</param>
+        /// <param name="task">The archive location on disk.</param>
```

Comment:
xmldoc is slightly stale with this change, although at this point might as well just cull it maybe as it's internal API now - but if not, how about
```suggestion
        /// <param name="task">The <see cref="ImportTask"/> containing data about the <typeparamref name="TModel"/> to import.</param>
```